the fetching process

```diff
@@ -1,11 +1,14 @@
+###Here you identified a story text you'd like to analyze. You will use the nltk library and use urllib to "fetch" the text. 
 >>> import nltk
 >>> from urllib import request
 >>> url = "http://classics.mit.edu/homer/iliad.mb.txt"
 >>> response = request.urlopen(url)
 >>> raw= response.read().decode('utf8')
 >>> type(raw)
 <class 'str'>
+###What are tokens? 
```

Comment:
### tokens are a basic component of source code that isolates selected words or phrases to be recalled later